agent/lib/kontena/observable.rb

```diff
@@ -10,6 +10,12 @@ def observable_value
       @observable_value
     end
 
+    # Obsevable has updated, as has not reset
+    # @return [Boolean]
+    def observable?
+      !!@observable_value
```

Comment:
This should probably also become false if this is an `ObservableObserver`, and one of the dependent observers gets reset.

Example: there's a node_info update with some invalid configuration that causes the weave restart to fail, so the `Kontena::Launchers::Weave` Observable resets itself into an error state. That reset will propagate to any Observers, but it just blocks the `observe` block from updating, and it does not cause those dependent observables to reset themselves. The `Kontena::NetworkAdapters::Weave` will still be `observable?`, so any `Kontena::ServicePods::Creator#network_adapter` -> `wait_until! { Actor[:network_adapter].observable? }` will proceed, but fail on the `Kontena::NetworkAdapters::Weave#reserve_container_address` (weave router is down -> etcd won't work -> IPAM will be broken). If the observable reset also propagated to dependent observables, then the service deploys would wait for the network adapter to be back up again... failing with timeouts on `network ready` instead of IPAM errors?